content/en/docs/spinnaker/using-dinghy.md

```diff
@@ -1097,3 +1256,223 @@ Delete all Webhook validations with the `--all` parameter.
 ```bash
 hal armory dinghy webhooksecrets <version control provider> delete --all
 ```
+
+
+## Application updates
+
+The Dinghy service behavior for applications and pipelines are different:
+- Applications: Dinghy creates the application with the configuration initially sent. The application does not get updated after that.
+- Pipelines: Updates on every push.
+
+However, you can change this behavior by adding a global variable: `save_app_on_update`. This behavior is good if you want to have your application to always adhere to the code implemented since it overwrites notifications every single time.
```

Comment:
I don't know if "implemented" here means the latest version of the code or the original version of the code. I suggest: 
" if you want to have your application adhere to the newest code, since it overwrites..." 
Unless I'm getting it backwards. :)